docs/modules/ROOT/pages/cicd_concept.adoc

```diff
@@ -0,0 +1,39 @@
+= Continuous Delivery
+
+In order to ensure that code released to production is of the highest possible quality, we recommend
+setting up three separate deployment environments to play different roles in the Continuous Integration 
+/ Continuous Delivery process.
+
+== Target Enviroments
+
+A typical CI/CD framework would deploy images to the following three environments:
```

Comment:
Below we call it "CI/CD process". Maybe it's a "workflow"?

Can we use consistent wording?

Reply:
I didn't mean for them to be two terms for the same thing.  The CI/CD framework is the infrastructure (eg. scripts on GitLab) that implements the CI/CD process.

Reply:
Correct. Good point!